cabal-install/Distribution/Client/CmdRepl.hs

```diff
@@ -335,16 +338,21 @@ data OriginalComponentInfo = OriginalComponentInfo
   }
   deriving (Show)
 
-withProject :: ProjectConfig -> Verbosity -> [String] -> IO (ProjectBaseContext, [TargetSelector], IO ())
+data ReplType = ProjectRepl | GlobalRepl FilePath
```

Comment:
please add some haddocks.

Is `GlobalRepl`'s `FilePath` argument intended working directory?

Reply:
Correct.